svc/IdentityService.ts

```diff
@@ -159,4 +160,11 @@ export class IdentityService extends HoistService {
         }
         return false;
     }
+
+    private canUserImpersonate(user: HoistUser): boolean {
+        const hasPerm = checkMinVersion(XH.environmentService.get('hoistCoreVersion'), '16.3.0')
```

Comment:
Nice fallback handling here across the HC release.

I think it would benefit us to offer a ready-made `checkMinHoistCoreVersion()`.